src/Lumper.UI/ViewModels/Bsp/Lumps/PakFile/PakFileEntryVtfViewModel.cs

```diff
@@ -23,198 +25,112 @@ public class PakFileEntryVtfViewModel : PakFileEntryLeafViewModel
         $"PakFileEntry{(string.IsNullOrWhiteSpace(Name) ? "" : $" ({Name})")}";
 
     private string _info = "";
+
     public string Info
     {
         get => _info;
         set => this.RaiseAndSetIfChanged(ref _info, value);
     }
 
     private bool _isModified = false;
-    public override bool IsModified { get => _isModified; }
+
+    public override bool IsModified
+    {
+        get => _isModified;
+    }
 
     public Image<Rgba32>? _image = null;
```

Comment:
should be private (my fault, but you've fixed the others)

Reply:
will do